src/global_distrib/mod_global_distrib.erl

```diff
@@ -43,15 +43,21 @@ start(Host, Opts0) ->
 stop(Host) ->
     mod_global_distrib_utils:stop(?MODULE, Host, fun stop/0).
 
--spec get_metadata(mongoose_acc:t(), Key :: term(), Default :: term()) -> Value :: term().
+-spec get_metadata(mongoose_acc:t(), Key :: term(), Default :: term()) ->
```

Comment:
Feels like it makes sense to start using `{ok, Value} | {error, Reason}` return format.

Reply:
Also, it makes sense to just return `Default` instead.

Reply:
Since there's no possibility for error I'd stick with returning a single value. AFAIK that's normal for functions taking a default (`proplists:get_value`, `map:get`, `application:get_env`).

Reply:
I wanted to keep consistent API for `get_` functions... so maybe we can rename them to `get_metadata/3` and `find_metadata/2`, so the API will be similar to `map` equivalents? Single value returned by `/3` and tuple or error for `/2`.

Reply:
Sure, go ahead :)